eda.Rmd

```diff
@@ -0,0 +1,213 @@
+---
+title: "eda"
+author: "Akansha Vashisth"
+date: '2019-04-03'
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r}
+library(ggplot2)
+library(tidyverse)
+library(lubridate)
+```
+
+### Survey question
+
+**How does distance from campus influence arrival time to lectures?**
+
+### Collected data
+```{r}
+# Initial survey data to go here
+initial_survey <- read_csv('data/Arrival Time_April 4, 2019_08.20.csv')
+```
+
+
+### Preprocessing on data
+```{r}
+# Ian please put your lovely code here :)
+
+clean_survey_sep_days <- initial_survey %>%
+  select(contains('Q')) %>%
+  slice(-c(1:2)) %>%
+  rename('distance_km' = 'Q1',
+         'mw_arrival' = 'Q3',
+         'tt_arrival' = 'Q4',
+         'mode_of_transport' = 'Q5') %>% 
+  mutate(mw_arrival = hm(mw_arrival),
+         mw_arrival = hour(mw_arrival)*60 + minute(mw_arrival),
+         mw_arrival = if_else(mw_arrival < 400, 540 + mw_arrival, mw_arrival),
+         mw_arrival = 9*60 - mw_arrival) %>%
+  mutate(tt_arrival = hm(tt_arrival),
+         tt_arrival = hour(tt_arrival)*60 + minute(tt_arrival),
+         tt_arrival = if_else(tt_arrival < 430, 570 + tt_arrival, tt_arrival),
+         tt_arrival = 9.5*60 - tt_arrival) %>%
+  mutate(mode_of_transport = fct_recode(mode_of_transport, 
+                                        driving = '1',
+                                        transit = '2',
+                                        walking = '3',
+                                        cycling = '4'))
+
+clean_survey_sep_days
+clean_survey_sep_days$distance_km <- as.numeric(clean_survey_sep_days$distance_km)
+
+
+View(clean_survey_sep_days)
+
+levels(clean_survey_sep_days$mode_of_transport)
+
+
+clean_survey_mw <- clean_survey_sep_days %>% 
+  select(-c(tt_arrival)) %>% 
+  rename(arrival = mw_arrival)
+
+clean_survey_tt <- clean_survey_sep_days %>% 
+  select(-c(mw_arrival)) %>% 
+  rename(arrival = tt_arrival)
+
+clean_survey_all_days <- bind_rows(clean_survey_mw, clean_survey_tt)
+```
+
+
+### Cleaned data
+
+```{r}
+survey_data <- read_csv("data/clean_survey_responses.csv")
+head(survey_data)
+```
```

Comment:
just a note: if we do EDA for separate days as well as aggregated days, we'll need to read 2 separate files in

Reply:
I think since it is EDA part of the project, we could show whatever we are doing(just to be explicit).
I have not yet uploaded the data. Just wanted to confirm before pushing both the csv files.